synapse/handlers/federation.py

```diff
@@ -478,6 +480,18 @@ async def do_invite_join(
                 partial_state=ret.partial_state,
             )
 
+            if ret.partial_state:
+                # kick off the process of asynchronously fixing up the state for this
+                # room
```

Comment:
"Fixing" makes me think it is broken, but it is really just incomplete, I *think*.

```suggestion
                # Kick off the process of asynchronously fetching the state for this
                # room.
```